spec/features/contact_centre_agent_creates_appointments_spec.rb

```diff
@@ -0,0 +1,69 @@
+# rubocop:disable Metrics/MethodLength
+require 'rails_helper'
+
+RSpec.feature 'Contact centre agent creates appointments' do
```

Comment:
Yep, that seems cool. Just had a chat with @mattlucht about it and it seems 'guiders', 'agents' and 'resource managers' are clear enough names.